actors/builtin/miner/monies_test.go

```diff
@@ -121,19 +122,25 @@ func TestPledgePenaltyForTermination(t *testing.T) {
 		dayReward := big.Div(initialPledge, bigInitialPledgeFactor)
 		oldDayReward := big.Mul(big.NewInt(2), dayReward)
 		twentyDayReward := big.Mul(dayReward, bigInitialPledgeFactor)
-		sectorAgeInDays := int64(20)
-		sectorAge := abi.ChainEpoch(sectorAgeInDays * builtin.EpochsInDay)
-		replacementAge := abi.ChainEpoch(15 * builtin.EpochsInDay)
+		oldSectorAgeInDays := int64(20)
+		oldSectorAge := abi.ChainEpoch(oldSectorAgeInDays * builtin.EpochsInDay)
+		replacementAgeInDays := int64(15)
+		replacementAge := abi.ChainEpoch(replacementAgeInDays * builtin.EpochsInDay)
 
 		// use low power, so termination fee exceeds SP
 		power := big.NewInt(1)
 
-		expectedFee := big.Div(big.Add(
-			big.Mul(big.NewInt(int64(sectorAge)), oldDayReward),
-			big.Mul(big.NewInt(int64(replacementAge)), dayReward)),
-			big.NewInt(builtin.EpochsInDay))
-
-		fee := miner.PledgePenaltyForTermination(dayReward, replacementAge, twentyDayReward, powerEstimate, power, rewardEstimate, dayReward, sectorAge)
+		oldPenalty := big.Div(
+			big.Product(oldDayReward, big.NewInt(oldSectorAgeInDays), miner.TerminationRewardFactorV2.Numerator),
+			miner.TerminationRewardFactorV2.Denominator,
+		)
+		newPenalty := big.Div(
+			big.Product(dayReward, big.NewInt(replacementAgeInDays), miner.TerminationRewardFactorV2.Numerator),
+			miner.TerminationRewardFactorV2.Denominator,
+		)
+		expectedFee := big.Sum(initialPledge, oldPenalty, newPenalty)
```

Comment:
This was tricky. As far as I can tell, the prior `expectedFee` calculation was wrong. Note the second-last parameter was `dayReward`, I changed it to `oldDayReward`. And how was the +initialPledge excluded? 

Does this look right to you @acruikshank

Reply:
Yes, the new code looks right. I would replace `initialPledge` with `twentyDayReward` for clarity in the final sum (they happen to be equal).

That the old version even passed appears to be an unlucky coincidence that sectorAgeInDays happened to equal the pledge factor and oldBR was chosen to be double day reward (which produced a difference between dayReward and oldDayReward equal to IP).